code/game/objects/items/explosives/grenades/bullet_grenade.dm

```diff
@@ -1,4 +1,4 @@
-///bullet grenade: shoots bullets out from itself in a ring
+  ///bullet grenade: shoots bullets out from itself in a ring
```

Comment:
Why did you touch this indentation.
Put it back where it was

Reply:
gdi fixing it now